csfieldguide/templates/interactives/algorithm-timer.html

```diff
@@ -0,0 +1,191 @@
+{% extends interactive_mode_template %}
```

Comment:
For really large results the 2 decimal places are still shown (regardless of unit)
![image](https://user-images.githubusercontent.com/25916475/57202399-cdfd1a80-6ff8-11e9-9313-85626960a1fa.png)

Reply:
As an aside, this is the largest possible result
![image](https://user-images.githubusercontent.com/25916475/57202547-a3ac5c80-6ffa-11e9-8e1b-304c726f7fd0.png)